plenum/server/node.py

```diff
@@ -1740,7 +1740,13 @@ def processRequest(self, request: Request, frm: str):
 
         ledgerId = self.ledgerIdForRequest(request)
         ledger = self.getLedger(ledgerId)
-        reply = self.getReplyFromLedger(ledger, request)
+
+        if request.operation[TXN_TYPE] == GET_TXN:
+            self.transmitToClient(RequestAck(*request.key), frm)
+            reply = self.handleGetTnxReq(request, frm)
+        else:
+            reply = self.getReplyFromLedger(ledger, request)
+
         if reply:
             logger.debug("{} returning REPLY from already processed "
```

Comment:
We log this message for GET_TXN. This looks incorrect, the GET_TXN is not yet processed. We should just transmit it to client.